CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Tracking/Models/GiniErrorAnalytics.swift

```diff
@@ -7,9 +7,9 @@
 import Foundation
 
 public struct GiniErrorAnalytics {
```

Comment:
[Suggestion]: @ValentinaIancu-Gini could you please add inline comment for `GiniErrorAnalytics`